CHANGELOG.md

```diff
@@ -10,7 +10,8 @@
 
 ### Additions and Improvements
 - Clean up old beacon states when switching from ARCHIVE to PRUNE or MINIMAL data storage mode
+- Upgrade to jvm-libp2p 1.2.1 which improve message publishing over gossipsub when flood publishing is disabled and IDONTWANT control message usage 
```

Comment:
I'd suggest making the changelog more friendly: 
```suggestion
- Upgrade to jvm-libp2p 1.2.1 to enable flood publishing based on max message size threshold
```
Not sure about the rephrasing but I feel it should be less into technical/implementation details

Reply:
Ye, point is that there are several improvements.
You're right though, going to revisit it